src/Components/SubscribeCard.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { useState } from 'react'
+import { useForm } from 'react-hook-form'
+import * as yup from 'yup';
+import {yupResolver} from '@hookform/resolvers/yup'
+import '../Styles/SubscribeCard.css'
+
+
+
+
+function SubscribeCard() {
+  const schema = yup.object().shape({
+    email: yup.string().email('Please provide a valid email address.').required('Please provide an email.')
+  })
+  
+
+  const [success, setSuccess] = useState('')
+  const { register, handleSubmit, formState: {errors}, reset} = useForm({
+    resolver: yupResolver(schema)
+  });
+
+  const onSubmit = (data) => {
+    console.log(data);
```

Comment:
logs should be removed

Reply:
Why is the log an issue ?? just trying to learn

Reply:
console logs are for development debugging. You don't include the in production code.